ui/packages/catalog-ui-search/src/main/webapp/react-component/container/table-export/table-export.tsx

```diff
@@ -249,15 +281,16 @@ export default hot(module)(
               exportFormat={this.state.exportFormat}
               exportSizeOptions={this.state.exportSizes}
               exportSize={this.state.exportSize}
-              handleExportFormatChange={this.handleExportFormatChange.bind(
-                this
-              )}
-              handleExportSizeChange={this.handleExportSizeChange.bind(this)}
-              onDownloadClick={this.onDownloadClick.bind(this)}
+              handleExportFormatChange={this.handleExportFormatChange}
+              handleExportSizeChange={this.handleExportSizeChange}
+              handleCustomExportCountChange={this.handleCustomExportCountChange}
+              onDownloadClick={this.onDownloadClick}
               warning={getWarning(
                 this.state.exportSize,
-                this.props.selectionInterface
+                this.props.selectionInterface,
```

Comment:
destructure props and state where it can help you use object shorthand

Reply:
Destructuring props and state to use object shorthand